xmtp_mls/src/identity_updates.rs

```diff
@@ -528,6 +529,43 @@ async fn verify_updates(
     .await
 }
 
+/// A static lookup method to verify if an identity is a member of an inbox
+pub async fn is_member_of_association_state<Client>(
+    api_client: &ApiClientWrapper<Client>,
+    inbox_id: &str,
+    identifier: &MemberIdentifier,
+) -> Result<bool, ClientError>
+where
+    Client: XmtpMlsClient + XmtpIdentityClient + ClientWithMetadata + Send + Sync,
+{
+    let filters = vec![GetIdentityUpdatesV2Filter {
+        inbox_id: inbox_id.to_string(),
+        sequence_id: None,
+    }];
+    let mut updates = api_client.get_identity_updates_v2(filters).await?;
+
+    let Some(updates) = updates.remove(inbox_id) else {
+        return Err(ClientError::Generic(
+            "Unable to find provided inbox_id".to_string(),
+        ));
+    };
+    let updates: Vec<_> = updates.into_iter().map(|u| u.update).collect();
+
+    let mut association_state = None;
+    let scw_verifier = RemoteSignatureVerifier::new(api_client.api_client.clone());
```

Comment:
Would be nice if the caller could specify the verifier, and for this to be the default if they don't.

Reply:
Done